ostree-ng.sh

```diff
@@ -485,22 +495,146 @@ greenprint "📥 Downloading the image"
 sudo composer-cli compose image "${COMPOSE_ID}" > /dev/null
 ISO_FILENAME="${COMPOSE_ID}-${INSTALLER_FILENAME}"
 modksiso "${ISO_FILENAME}" "/var/lib/libvirt/images/${ISO_FILENAME}"
-sudo rm -f "${ISO_FILENAME}"
 
 # Clean compose and blueprints.
 greenprint "🧹 Clean up compose"
 sudo composer-cli compose delete "${COMPOSE_ID}" > /dev/null
 sudo composer-cli blueprints delete installer > /dev/null
 
+##################################################################
+##
+## Install edge vm with edge-installer (http boot)
+##
+##################################################################
+
+if [[ "${HTTP_BOOT_FEAT}" == "true" ]]; then
+
+    greenprint "📋 Mount installer iso and copy content to webserver/httpboot"
```

Comment:
Could you please move them into a function? That makes the script read-able. Thanks.